website/screens/components/button/specs/ButtonSpecsPage.tsx

```diff
@@ -1,855 +1,915 @@
+              <td>
+                <Code>small</Code>
+              </td>
+              <td>60px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>medium</Code>
+              </td>
+              <td>240px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>large</Code>
+              </td>
+              <td>480px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>fillParent</Code>
+              </td>
+              <td>-</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>fitContent</Code>
+              </td>
+              <td>-</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <DxcText as="p">
+          The component <Code>height</Code> is fixed:
+        </DxcText>
+        <DxcTable>
+          <thead>
+            <tr>
+              <th>height</th>
+              <th>value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <Code>min-height</Code>
+              </td>
+              <td>40px</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <HeadingLink level={4}>Margin</HeadingLink>
+        <DxcText as="p">
+          Button <Code>margin</Code> can use the values:
+        </DxcText>
+        <DxcTable>
+          <thead>
+            <tr>
+              <th>margin</th>
+              <th>value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <Code>xxsmall</Code>
+              </td>
+              <td>6px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>xsmall</Code>
+              </td>
+              <td>16px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>small</Code>
+              </td>
+              <td>24px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>medium</Code>
+              </td>
+              <td>36px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>large</Code>
+              </td>
+              <td>48px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>xlarge</Code>
+              </td>
+              <td>64px</td>
+            </tr>
+            <tr>
+              <td>
+                <Code>xxlarge</Code>
+              </td>
+              <td>100px</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <DxcText as="p">
+          These values can be applied independently to each side of the
+          component: <Code>top</Code>, <Code>bottom</Code>, <Code>left</Code>,
+          <Code>right</Code>.
+        </DxcText>
+        <HeadingLink level={5}>Typography</HeadingLink>
+        <DxcTable>
+          <thead>
+            <tr>
+              <th align="left">Component token</th>
```

Comment:
I don't think align="left" because that property comes from the th of the table and it is overriding this one. Same for td.